src/super_gradients/training/transforms/obb/obb_sample.py

```diff
@@ -0,0 +1,110 @@
+import dataclasses
+from typing import Union, List, Optional
+
+import numpy as np
+import torch
+from super_gradients.training.datasets.data_formats.obb.cxcywhr import cxcywhr_to_poly, poly_to_cxcywhr
+
+
+@dataclasses.dataclass
+class OBBSample:
```

Comment:
We have a dedicated src/super_gradients/training/samples module which is more appropriate to place this in. Please move it there.

Reply:
Thanks! I forgot we have this folder :/
Fixed in 874119c2f1f7e2855eff8c8b9e07518223f90317